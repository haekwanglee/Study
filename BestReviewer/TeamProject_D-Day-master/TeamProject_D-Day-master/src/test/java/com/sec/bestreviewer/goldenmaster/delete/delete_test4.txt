DEL,-p, , ,certi,ADV
SCH,-p, , ,certi,ADV
DEL,-p, , ,certi,EX
SCH,-p, , ,certi,EX