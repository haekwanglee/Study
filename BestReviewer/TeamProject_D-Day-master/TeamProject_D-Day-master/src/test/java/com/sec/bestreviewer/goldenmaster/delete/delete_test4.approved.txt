DEL,90000000,FIRST MAN,CL2,010-1111-1111,19980101,ADV
DEL,90000001,AAAAAA AAA,CL2,010-1111-1112,19980102,ADV
DEL,91000000,AAAAAAA AAA,CL2,010-1111-1113,19980103,ADV
DEL,91000001,AAAAA BBB,CL2,010-1111-1114,19980104,ADV
DEL,92000000,AAAAA CCC,CL2,010-1111-1115,19980105,ADV
SCH,NONE
DEL,49000001,CERTIEX BABY,CL2,010-5555-1118,19931001,EX
DEL,50000001,CERTIEX BABY,CL2,010-5555-1119,19931001,EX
DEL,89000001,CERTIEX BABY,CL2,010-5555-1110,19931001,EX
DEL,89002001,CERTIEX BABY,CL2,010-5555-1120,19931001,EX
DEL,89032001,CERTIEX BABY,CL2,010-5555-1130,19931001,EX
SCH,NONE
